### ISSUE TITLE
Workspace environment

### ISSUE THREAD

**Issue by StjepanBM1** [ringwormGO-organization/ringOS#27]
**TL;DR** A lot of ringwormGO projects are rushed because of unrealistic deadlines (cutting corners included).

As a Project Director of ringwormGO I noticed there is a lot of toxicity around here. For example -> a lot of projects are rushed because of this they  don't have that bit of quality that is expected. Most people working on ringwormGO usually are worikng **3 separate projects at once**, as a result there is a lot of pressure to hit deadlines (even though they are unrealistic), meaning that projects often need a lot of updates after the release (don't get me started about us cutting corners).

What do you think?

**Comment by Andrej123456789** [comment]
This is for ringwormGO repo.

I will transfer it
